MeNext/net/API.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Xamarin.Forms;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Runtime.Serialization.Json;
+
+namespace MeNext
+{
+
+	public class API
+	{
+		public API(string ip)
+		{
+			this.ServerIP = ip;
+			this.Client = new HttpClient();
+
+		}
+
+		/// <summary>
+		///  SayHello is a testing method that just servers to ping the server. 
+		/// </summary>
+		/// <returns>The server response string.</returns>
+		public async System.Threading.Tasks.Task<string> SayHello()
+		{
+			var uri = new Uri(this.ServerIP + "/hello");
+			return await FireRequest(uri);
```

Comment:
Don't `await` here. Just return the `Task<string>`. This allows the caller to `SayHello()`, do stuff, and then `await` the result later. It also allows it to simply fire the request and not wait for a response (_ex_ we might want to do this for pause).

Only writing this once. Applies to similar methods.

See https://msdn.microsoft.com/en-us/library/mt674882.aspx for a "quick" tut on async/await